ReportingServicesTools/ReportingServicesTools.psd1

```diff
@@ -91,7 +91,12 @@
         'Set-RsEmailSettings',
         'Set-RsUrlReservation'
         'Write-RsCatalogItem',
-        'Write-RsFolderContent'
+        'Write-RsFolderContent',
+	'Get-RsSubscription',
```

Comment:
please use spaces for the space, align it with the existing list and remove the empty lines